kuka_kr210_support/urdf/kr210_variants_macro.xacro

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
I would move this code into the kr210l150_macro.xacro and then delete this file.  The `variants` naming is confusing here.

Reply:
Done. Committed/pushed. I had a tricky time figuring out how to get at the `arg` from the top level file. From looking around `xacro:property` seemed like the ticket, so that's what I used. Hopefully that's correct? I admit to not knowing precisely why `arg` vs. `param` vs. `property`...

```
  <!-- for the KR210 L150, use ""; for the KR210 L150-2, use "_2" -->
  <xacro:arg name="variant" default="" />
  <xacro:property name="variant" value="$(arg variant)"/>
```